### PR TITLE
#134345313  Environment variable setup

### PR DESCRIPTION
**Pull request by andela-emurphy**
#### What does this PR do?
 Set up environment variables for  production, test, and development.             
#### Description of Task to be completed?
 Separates local development , test and production details. Hides important details
about server configurations  e.g database url      .
#### How should this be manually tested?  
- Change the .env.example to .env
- Fill in the appropriate details
- Start your server

### CODE REVIEW

**Comment by andela-kakpobome** [comment]
@andela-emurphy you must take care of all these issues before this PR would be merged.

**Comment by andela-kakpobome** [comment]
@andela-emurphy there's still a merge conflict here.

File: test/client/test.spec.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
-'use strict';
-
 function square(x) {
   return x * x;
 }
 
-describe('simple test ' , function() {
-  it('should return square of 4', function() {
+describe('simple test<div></div> ' , function() {
```
Comment (adebayo-maborukoje) [inline test/client/test.spec.js]: i don't understand what this `<div></div>` is doing here. @andela-emurphy

File: test/client/test.spec.js [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
+  it('should return sqrt ', function() {
```
Comment (adebayo-maborukoje) [inline test/client/test.spec.js]: this should read `should return square`

File: test/client/test.spec.js (hunk 1 above)
Comment (andela-emurphy) [inline test/client/test.spec.js]: fixed

File: test/client/test.spec.js (hunk 2 above)
Comment (andela-emurphy) [inline test/client/test.spec.js]: fixed

File: server.js [hunk 3]
```diff
@@ -55,12 +56,12 @@ require('./config/express')(app, passport, mongoose);
...
 //Start the app by listening on <port>
-var port = config.port;
-var server = app.listen(port);
+
+var server = app.listen(config.port;);
 var ioObj = io.listen(server, { log: false });
 //game logic handled here
 require('./config/socket/socket')(ioObj);
-console.log('Express app started on port ' + port);
+console.log('Express app started on port ' + config.port;);
```
Comment (andela-emurphy) [inline server.js]: fixed

File: server.js [hunk 4]
```diff
@@ -55,12 +56,12 @@ require('./config/express')(app, passport, mongoose);
 require('./config/routes')(app, passport, auth);
 
 //Start the app by listening on <port>
-var port = config.port;
-var server = app.listen(port);
+
+var server = app.listen(config.port;);
```
Comment (andela-emurphy) [inline server.js]: fixed

File: jasmine/SpecRunner.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
```
Comment (andela-emurphy) [inline jasmine/SpecRunner.html]: fixed

**Pull request merged**